openjdk.build/makefile

```diff
@@ -141,6 +146,13 @@ $(warning STF_ROOT is $(STF_ROOT))
 PREREQS_ROOT=$(OPENJDK_SYSTEMTEST_ROOT)$(D)..$(D)..$(D)systemtest_prereqs
 $(warning PREREQS_ROOT is $(PREREQS_ROOT))
 
+# Add JCK_ROOT to the PREREQS_ROOT directories if one has been provided on the command line
```

Comment:
At the moment this requires the directory to be one level up from where it may be intuitive i.e. the directory specified needs to be `$JCK_ROOT/jck/jck8b` whereas it would seem more logical from a user-perspective not to require two levels underneath there. If we completely disassociate the jck from the "prereqs" then we can probably improve this

Reply:
Well, I could default JCK_ROOT to systemtest_prereqs/jck and then if you overrode it you could just specify JCK_ROOT=<where-you-put-the-jck>.  The jck subdirectory anticipates there being multiple versions of the JCK on the test machine and that we would want to see them under a common directory.  Do you think that would be a solution?

Reply:
SGTM

Reply:
I've changed the STF automation to look for the jckVersion directory (jck8b, jck9 etc) directly beneath the -systemtest-prereqs directories or under a /jck subdirectory.  If there are multiple copies then a /jck/jckVersion 'wins'.  Within that, the leftmost -systemtest-prereqs directory will win.